### ISSUE TITLE
Getting no pixel values in mask.

### ISSUE THREAD

**Issue by sansinghsanjay** [raghakot/ultrasound-nerve-segmentation#3]
Hi,

I changed the dimension ordering of keras to 'theano' with backend 'tensorflow', dropped all images in "input" folder and then ran your code. Code is executing without any error but I am not getting any pixel value (mask) for test set.

Even I ran your code after commenting line {65, 66} of file submission.py:
#if has_masks[i, 0] < 0.5:
     #masks[i, 0] *= 0.

But still there is no pixel value (mask) for any image of test set. Please help me..

Thanks...

**Comment by sansinghsanjay** [comment]
Hi,

I tried various things in your code and found that line no 58 of data.py should be like this:

imgs = np.ndarray((total, DataManager.IMG_ORIG_ROWS, DataManager.IMG_ORIG_COLS), dtype=np.uint8)

instead of:

imgs = np.ndarray((total, **1,** DataManager.IMG_ORIG_ROWS, DataManager.IMG_ORIG_COLS), dtype=np.uint8)

So, finally this solves my issue.

Thanks....
